src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,29 @@
-export const PostInfo = () => <>Put the post here</>;
+import { CommentList } from '../CommentList';
+import './PostInfo.scss';
+import { UserInfo } from '../UserInfo';
+
+export const PostInfo = ({ post }) => {
+  const { title, user, body, comments } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">
+          {title}
+        </h3>
+
+        <p>
+          {' Posted by  '}
```

Comment:
It would be better to move this text inside userInfo, since it logically belongs there. We should see `Posted by ${name}` in the component so it is predictable how it's gonna look